### PR TITLE
rate-comment

### PR DESCRIPTION
**Pull request by tungblvt**
==> Title: Rate and comment store

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/13590

### CODE REVIEW

**Comment by tungblvt** [comment]
ready

**Comment by tungblvt** [comment]
ready

File: app/helpers/comments_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```
Comment (huuphuc93) [inline app/helpers/comments_helper.rb]: xóa file này đi nếu k dùng

File: app/views/comments/_list.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<div class="wrap-comment">
+  <span class="wrap-thumb">
+    <%= image_tag comment.user_avatar.url if comment.user_avatar %>
+  </span>
+  <span class="user"><%= comment.user_name %></span>
+  <span class="rated">
+    <% comment.rate.times do %>
+      <i class="fa fa-star active"></i>
+    <% end %>
+    <% (5 - comment.rate).times do %>
```
Comment (huuphuc93) [inline app/views/comments/_list.html.erb]: 5 -> Settings

**Comment by tungblvt** [comment]
ready

**Pull request merged**